docs/api/python.md

```diff
@@ -60,28 +60,46 @@ print(con.fetchall())
 > Do *not* use `executemany` to insert large amounts of data into DuckDB. See below for better options.
 
 ## Efficient Transfer
-Transferring large datasets to and from DuckDB uses a separate API built around [NumPy](https://numpy.org) and [Pandas](https://pandas.pydata.org). This API works with entire columns of data instead of scalar values and is therefore far more efficient. 
+Transferring large datasets to and from DuckDB uses a separate API built around [NumPy](https://numpy.org) and [Pandas](https://pandas.pydata.org), or [Apache Arrow](https://arrow.apache.org/). This API works with entire columns of data instead of scalar values and is therefore far more efficient. 
 
-DuckDB supports "registering" a Pandas data frame as a virtual table, comparable to a SQL `VIEW`. Below is an example:
+By default, DuckDB will automatically be able to query any variable that is a Pandas or Arrow data frame by name. DuckDB also supports "registering" a data frame as a virtual table, comparable to a SQL `VIEW`. This is useful when querying a data frame that is stroed in another way (as a class variable, or a value in a dictionary) Below is a Pandas example:
```

Comment:
I'm not 100% sure we can scan any arrow variable, for sure we can do datasets (memory and from disk), tables, recordbatchreader and scanners.
stroed ==> stored